extensions/ql-vscode/package.json

```diff
@@ -314,10 +314,6 @@
         "command": "codeQL.exportVariantAnalysisResults",
         "title": "CodeQL: Export Variant Analysis Results"
       },
-      {
```

Comment:
Why does this not provide value any more? It would allow someone (e.g. Design) to load the view without having actually ran a variant analysis. Is it not possible to keep this in a way that it loads a mock variant analysis?

Reply:
As it is currently, we would need to implement special handling for a mock view in the `VariantAnalysisView`. The mock view also only implements 1 specific variant analysis, so I think using #1577 would be more useful for someone to view a variant analysis, or using Storybook. I would be happy to add it back, but that will result in some additional code being added to `VariantAnalysisView` for handling a mock view (especially in `onWebViewLoaded`), or there would be some specific code for it in `extension.ts`.

Reply:
Let's leave it then - forgot about Storybook!